blocks/api/raw-handling/special-comment-converter.js

```diff
@@ -19,10 +19,27 @@ const { COMMENT_NODE } = window.Node;
  * @return {void}
  */
 export default function( node ) {
-	if (
-		node.nodeType !== COMMENT_NODE ||
-		node.nodeValue.indexOf( 'more' ) !== 0
-	) {
+	if ( node.nodeType !== COMMENT_NODE ) {
+		return;
+	}
+
+	if ( node.nodeValue.indexOf( 'nextpage' ) === 0 ) {
```

Comment:
I don't believe `nextpage` accepts any "arguments" within the comment, so, in order to match core, this should be a full string eq check: `node.nodeValue === 'nextpage'`.